draft-linus-trans-gossip-ct.md

```diff
@@ -424,7 +428,7 @@ outlined in {{privacy-SCT}} and {{privacy-trusted-auditors}}.
 
 The most natural trusted auditor arrangement arguably is a web browser
 that is "logged in to" a provider of various internet
-services. Another equivalent arrangement is a trusted party like a
+services [not entirely clear what you mean here]. Another equivalent arrangement is a trusted party like a
```

Comment:
I actually didn't view it as a logged in Chrome user.  Although that's another example.  

The example I was envisioning doesn't work perfectly though. I was thinking a client that's sitting on one ISP connection for it's lifetime. It might share the SCTs with its ISP.  Doesn't work well with laptops though. Although the data should be sent as part of a DHCP response...